infra/cifuzz/config_utils_test.py

```diff
@@ -39,3 +39,14 @@ def test_base_ref(self):
     """Tests that base_ref is set properly."""
     expected_base_ref = 'expected_base_ref'
     os.environ['GITHUB_BASE_REF'] = expected_base_ref
+    config = self._create_config()
+    self.assertEqual(config.base_ref, expected_base_ref)
+
+  def test_keep_unaffected_defaults_to_false(self):
+    """Tests that keep_unaffected_fuzz_targets defaults to false."""
+    config = self._create_config()
+    self.assertFalse(config.keep_unaffected_fuzz_targets)
```

Comment:
Do we test anywhere that config.keep_unaffected_fuzz_targets is True?

Reply:
I think testing the default behavior makes sense. Testing other behaviors like setting true or false not so much since were just testing getenv at that piont.